src/js/view/view.js

```diff
@@ -636,6 +636,10 @@ function View(_api, _model) {
         return provider.getFullScreen();
     }
 
+    function _nativeFullscreenChangeHandler(event) {
+        toggleClass(_playerElement, 'jw-flag-ios-fullscreen', event.jwstate);
```

Comment:
Do we need to do something like this?
`const newState = (event.jwstate !== undefined) ? event.jwstate : _isNativeFullscreen();`
I see that the event is coming from the provider and it is always adding `jwstate`, but the `_fullscreenChangeHandler` below gets the newState like that. Am I missing something?

Reply:
You should only add the class when `jwstate` is `true`, and the os is `ios`.

Reply:
`NATIVE_FULLSCREEN` event only fires with `event.jwstate = true` if it is iOS fullscreen.
This is the only place where `NATIVE_FULLSCREEN` is fired: (https://github.com/jwplayer/jwplayer/blob/master/src/js/providers/html5.js#L670)

Reply:
You think so, but it’s not guaranteed. For that reason, you should verify the iOS.